repos/spack_repo/builtin/packages/gtkplus/package.py

```diff
@@ -24,6 +24,8 @@ class Gtkplus(AutotoolsPackage, MesonPackage):
         default="autotools",
```

Comment:
Let's just use the default `meson` and see if that gets (I think) `e4s` stacks to use the newer versions.
```suggestion
        default="meson",
```